Lib/test/test_type_aliases.py

```diff
@@ -142,7 +142,16 @@ def test_subscripting(self):
 
     def test_repr(self):
```

Comment:
Could you also add some tests to `test_genericalias.py`? This file is primarily intended to test the `type` statement, not `types.GenericAlias`.

Reply:
Is it possible to create a pure and sensible `types.GenericAlias` with `ParamSpec` without a `type` statement? I cannot find any good examples.

Reply:
Not sure you could count this as sensible:

```
>>> class X:
...     __class_getitem__ = classmethod(types.GenericAlias)
... 
>>> X[[int, str]]
__main__.X[[<class 'int'>, <class 'str'>]]
```

Reply:
Sounds good! 👍